js/script.js

```diff
@@ -1,9 +1,33 @@
 import { $createButton, $taskContainer } from "./element.js";
 import { sanitizeInput, getUniqueId } from "./utlity.js";
+import { READ, EDIT } from "./const.js";
 
 let isFormOpen = false;
 let tasks = [];
 
+const editForm = (liElement, task) => {
+  const $inputElement = document.createElement("input");
+  const $updateButton = document.createElement("button");
+  const $cancelButton = document.createElement("button");
+  const $deleteButton = document.createElement("button");
+
+  $inputElement.setAttribute("id", `input-${task.id}`);
+
+  $inputElement.value = task.title;
+
+  $deleteButton.innerText = "Delete";
+  $updateButton.innerHTML = "Update";
+  $cancelButton.innerHTML = "Cancel";
+
+  $deleteButton.addEventListener("click", () => deleteTaskHandler(task.id));
+  $updateButton.addEventListener("click", () => updateTaskHandler(task.id));
+  $cancelButton.addEventListener("click", () => calcelTaskHandler(task.id));
```

Comment:
you can simply pass the task object instead of the task id and change its mode property because the task object will be passed as reference. In that case you do not need to loop through the tasks list to find the task.